packages/config/config/devices/0x001d/zw4sf.json

```diff
@@ -0,0 +1,110 @@
+// Leviton ZW4SF
+// 4 Speed Fan Controller
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "ZW4SF",
+	"description": "4 Speed Fan Controller",
+	"devices": [
+		{
+			"productType": "0x0038",
+			"productId": "0x0002",
+			"zwaveAllianceId": 3832
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"3": {
+			"label": "Min Level",
+			"valueSize": 1,
+			"minValue": 0,
```

Comment:
These should be unit "%". Carey through to the other 0-99 level ones. Put the unit directly under valueSize.